braze/client.py

```diff
@@ -259,3 +259,47 @@ def campaign_trigger_schedule_create(
             payload["recipients"] = recipients
 
         return self.__create_request(payload)
+
+    
+
+    def list_email_templates():
+        r = requests.get(endpoint + "/templates/email/list?api_key=" + api_key)
+        template_name = extract_values(r.json(), "template_name")
+        # email_template_id = extract_values(r.json(), "email_template_id")
+
+        pprint(template_name)
+        # pprint(email_template_id)
+        # pprint(r.text)
+
+    def create_new_email(email_name, subject):
+        email_name = input("Name of email: ")
```

Comment:
If email_name is given as arg why take it as an input as well? I think just drop the `input` calls just the method args are fine.